server/pipeline/create.go

```diff
@@ -87,13 +87,13 @@ func Create(ctx context.Context, _store store.Store, repo *model.Repo, pipeline
 
 		return nil, ErrFiltered
 	} else if configFetchErr != nil {
-		log.Debug().Str("repo", repo.FullName).Err(configFetchErr).Msgf("error while fetching config '%s' in '%s' with user: '%s'", repo.Config, pipeline.Ref, repoUser.Login)
+		log.Error().Str("repo", repo.FullName).Err(configFetchErr).Msgf("error while fetching config '%s' in '%s' with user: '%s'", repo.Config, pipeline.Ref, repoUser.Login)
 		return nil, updatePipelineWithErr(ctx, _forge, _store, pipeline, repo, repoUser, fmt.Errorf("could not load config from forge: %w", err))
 	}
 
 	pipelineItems, parseErr := parsePipeline(_forge, _store, pipeline, repoUser, repo, forgeYamlConfigs, nil)
 	if pipeline_errors.HasBlockingErrors(parseErr) {
-		log.Debug().Str("repo", repo.FullName).Err(parseErr).Msg("failed to parse yaml")
+		log.Error().Str("repo", repo.FullName).Err(parseErr).Msg("failed to parse yaml")
```

Comment:
It would not work well on public instance, I think.

Reply:
Not sure I follow. Something about the risk of someone flooding the logs?

Reply:
> the risk of someone flooding the logs

Yes. While error on pulling config (whether config server / forge is misconfigured or network issues) is something admin copes with, pipeline syntax error is 'users' mistake.